CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * [#7639](https://github.com/rubocop-hq/rubocop/pull/7639): Fix logical operator edge case in `omit_parentheses` style of `Style/MethodCallWithArgsParentheses`. ([@gsamokovarov][])
 * [#7661](https://github.com/rubocop-hq/rubocop/pull/7661): Fix to return correct info from multi-line regexp. ([@Tietew][])
+* [#7647](https://github.com/rubocop-hq/rubocop/issues/7647): Handle numblock. ([@hanachin][])
```

Comment:
As @pocke mentions at https://github.com/rubocop-hq/rubocop/pull/7665#issuecomment-578123784, this patch doesn't solve all known issue of `numblock`. So the `Handle numblock` seems a bit broad sense to me. What about alternatives like the following message focused on resolving the error?

```diff
-* [#7647](https://github.com/rubocop-hq/rubocop/issues/7647): Handle numblock. ([@hanachin][])
+* [#7647](https://github.com/rubocop-hq/rubocop/issues/7647): Fix an `undefined method on_numblock` error when using Ruby 2.7's numbered parameters. ([@hanachin][])
```